app/default/account_routes.py

```diff
@@ -92,6 +83,12 @@ def build_application_data_for_display(applications: list[ApplicationSummary]):
 @login_required
 def dashboard():
     account_id = g.account_id
+
+    # fund_short_name = request.args.get("fund")
+    # round_short_name = request.args.get("round")
```

Comment:
Do we want to keep these commented lines?

Reply:
Yeah, they are kind of a reminder to me for the next part in all this with the dashboard if that's ok